src/main/java/io/pravega/perf/PravegaStreamHandler.java

```diff
@@ -162,7 +167,7 @@ void deleteReaderGroup() {
         try {
             readerGroupManager.deleteReaderGroup(rdGrpName);
         } catch (RuntimeException e) {
-            System.out.println("Cannot delete reader group " + rdGrpName + " because it is already deleted");
+            log.info("Cannot delete reader group {} because it is already deleted", rdGrpName);
```

Comment:
This PR is fine but I think we shouldn't be catching this exception. deleteReaderGroup should not throw an exception if the stream does not exist.

Reply:
Agreed, in fact I don't even think `deleteReaderGroup` is even called anywhere.